app/databaseAccessors/photoTagAccessor.js

```diff
@@ -19,14 +32,14 @@ export default class PhotoTagAccessor {
   }
 
   /**
-   * Find tag by its ID
+   * Find and returns populated tag by its ID
    *
    * @param {ObjectID} tagID - The ID of the tag
-   * @returns Tag
+   * @returns Tag with populated field
    */
   static async getTagByID(tagID) {
     await Connection.open();
-    const tag = await PhotoTag.findById(new mongoose.Types.ObjectId(tagID));
+    const tag = await PhotoTag.findById(new mongoose.Types.ObjectId(tagID)).populate("creatingUser");
```

Comment:
I think for consistency we say `Id`, so `getTagById` instead, and call the parameter `tagId`. But you can fact check me on that